src/coreclr/vm/prestub.cpp

```diff
@@ -2367,6 +2367,13 @@ PCODE MethodDesc::DoPrestub(MethodTable *pDispatchingMT, CallerGCMode callerGCMo
     {
         _ASSERTE(!MayHaveEntryPointSlotsToBackpatch()); // This path doesn't lock the MethodDescBackpatchTracker as it should only
                                                         // happen for jump-stampable or non-versionable methods
+#ifdef FEATURE_PORTABLE_ENTRYPOINTS
+        if (pMT->IsDelegate())
+        {
+            return pCode;
```

Comment:
If we early out here, does it mean that we will be going through PreStub every time the delegate constructor is called? We should not be going through PreStub in steady state as we have discussed...